test/integration/apis/btc-core.test.ts

```diff
@@ -76,27 +76,6 @@ describe("BTCCore testnet", function () {
             assert.deepEqual(raw_tx, raw);
         });
     });
-
-    describe("getTxByOpcode", () => {
```

Comment:
for these integration tests, could you update the btc-core API to point to testnet? I assume those are deleted because the tests are not passing with using the local electrs instance in the docker container.